x/paloma/keeper/keeper_integration_test.go

```diff
@@ -56,7 +54,11 @@ var _ = Describe("jailing validators with missing external chain infos", func()
 			// val[1] has only one chain
 			// val[2] doesn't have anything
 			// All other vals have everything
-			err := a.ValsetKeeper.AddExternalChainInfo(ctx, vals[0].GetOperator(), []*valsettypes.ExternalChainInfo{
+			valAddress, err := utilkeeper.ValAddressFromBech32(k.AddressCodec, vals[0].GetOperator())
+			if err != nil {
```

Comment:
Okay, it's obvious you just copy/pasted this now. We don't need logging in test files. Just assert the error is `nil` and move on.

Reply:
Added assertion for error to be Nil